tests/test_machine.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import os
+from subprocess import TimeoutExpired
+
+import pytest
+
+import mkosi.machine as machine
+from mkosi.backend import MkosiException
+
+
+@pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
+@pytest.mark.parametrize("verb", ["boot", "qemu"])
+@pytest.mark.integration
+
+class TestVerbs:
```

Comment:
Bc then I didn't need to repeat pytest.mark.parametrize and mark.integration for every function. Will remove the class then

Reply:
Ahah, OK that makes sense. I had a look at the documentation and you can also define the `pytestmark` global variable which will do the same for all tests in this module. Since this is something we currently want for all tests, I'd just use that for now.